Rakefile

```diff
@@ -14,14 +15,19 @@ begin
   end
 
   task :mdl do
-    puts 'Running Markdown Lint...'
-    system 'mdl *.md' || exit(-1)
+    puts 'Linting markdown documents...'
+    MarkdownLint.run(Dir['*.md'])
+  # MarkdownLint#run calls system exit regardless of status.
+  rescue SystemExit # rubocop:disable Lint/SuppressedException
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Do not suppress exceptions.](https://app.codacy.com/manual/aaronmallen/activeinteractor/pullRequest?prid=4767422)